src/test-helpers/mock-dataset.json

```diff
@@ -0,0 +1,274 @@
+[
```

Comment:
As per previous comments, since this is a mock of GeoJSON features rather than a `dataset` in Hub's traditional sense, I'd rename this to be `mock-geojson-features` (or something of the like)

Reply:
That's a good point! I agree. I have renamed it to `mock-geojson-features`.